core/src/main/scala/net/ruippeixotog/scalascraper/browser/JsoupBrowser.scala

```diff
@@ -70,7 +75,7 @@ class JsoupBrowser(val userAgent: String = "jsoup/1.8", val proxy: JavaProxy = n
     new JsoupBrowser(userAgent, newJavaProxy)
   }
 
-  def requestSettings(conn: Connection): Connection = conn
+  private def requestSettings(conn: Connection): Connection = conn
```

Comment:
This is a breaking change, and not desirable - people should be able to extend `JsoupBrowser` and override this method to customize connection building.

You could make it protected... but it's still a breaking change and there doesn't seem to be a strong need for it here. Let's keep it public.

Reply:
done